src/shared/Core/Git.cs

```diff
@@ -219,6 +225,35 @@ public void AddFile(string filePath)
             }
         }
 
+        public void Commit(string message)
```

Comment:
Is there an opportunity to share some of this code with the `Add` method? May not be worthwhile, but was just a thought I had.

Reply:
I can understand